docs/program/apis/data-client.md

```diff
@@ -74,6 +74,66 @@ The data client API supports the following methods (among [others](https://pytho
 
 {{< readfile "/static/include/services/apis/data-client.md" >}}
 
+### TabularDataByFilter
+
+Retrieve optionally filtered tabular data from the [Viam app](https://app.viam.com).
+You can also find your tabular data under the **Sensors** subtab of the app's [**Data** tab](https://app.viam.com/data).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `filter` [(Optional[viam.proto.app.data.Filter])](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter): Optional `Filter` specifying tabular data to retrieve. Specify no filter to download all tabular data.
+- `dest` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Filepath to write retrieved data to.
```

Comment:
Might be worth mentioning that if not populated here or in BinaryDataByFilter, writes to your current directory.

Reply:
Sounds good, done! Thanks for pointing that out!